src/rpc/mining.h

```diff
@@ -13,6 +13,6 @@
 UniValue generateBlocks(std::shared_ptr<CReserveScript> coinbaseScript, int nGenerate, uint64_t nMaxTries, bool keepScript);
 
 /** Check bounds on a command line confirm target */
-unsigned int ParseConfirmTarget(const UniValue& value);
+unsigned int ParseConfirmTarget(const UniValue& value, unsigned int max_target);
```

Comment:
Wow. This should be moved into `rpc/util` (in a future PR). (also comment is outdated!)